src/config/dynamic-theme-fixes.config

```diff
@@ -5344,7 +5344,7 @@ INVERT
 .social-share-view_discovery-small
 .social-share-view_discovery-large
 .orgpage-social-links-view__icon
-.map-container > ymaps > ymaps > ymaps > ymaps
+.map-container > ymaps > ymaps > canvas
```

Comment:
Add it as I still don't have a canvas. I had this problem with Google maps also that Some experience on another location a canvas.

Reply:
@Gusted do you have `canvas` now? Since at the moment the old rule breaks the "Satellite" map mode for me.

Reply:
Yes I have the canvas now